presto-hive/src/main/java/io/prestosql/plugin/hive/HiveFlavor.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive;
+
+public enum HiveFlavor
+{
+    UNKNOWN,
+    AWS_GLUE,
+    FILE_METASTORE,
+    HDP2,
+    HDP3,
+    AWS_EMR,
```

Comment:
move it next to `GLUE`

Reply:
EMR is hadoop version.
GLUE is something totally different